integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/ImageCatalogTest.java

```diff
@@ -398,5 +402,76 @@ public void testGetListOfImageCatalogs(TestContext testContext) {
                 })
                 .validate();
     }
-    //TODO test get images from catalog for existing Stack
+
+    @Test(dataProvider = TEST_CONTEXT_WITH_MOCK)
+    public void testGetImageCatalogByNameAndStackWhenPreWarmedImageHasBeenUsed(TestContext testContext) {
+        createDefaultCredential(testContext);
+        initializeDefaultClusterDefinitions(testContext);
+
+        String imgCatalogName = getNameGenerator().getRandomNameForResource();
+        String stackName = imgCatalogName + "-stack";
+        MockedTestContext mockedTestContext = (MockedTestContext) testContext;
+
+        testContext
+                .given(ImageCatalogDto.class).withName(imgCatalogName).withUrl(mockedTestContext.getImageCatalogMockServerSetup().getPreWarmedImageCatalogUrl())
```

Comment:
'ImageCatalogDto' should be named as ImageCatalogTestDto

Reply:
:D Haha it's too late, but will rename it if there will be modifications around there.

Reply:
no need for it, I've done it my branch

Reply:
thanks